dev/ci/docker/README.md

```diff
@@ -4,10 +4,32 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-Gitlab CI will build and tag a Docker by default for every job if the
-`SKIP_DOCKER` variable is not set to `false`. In Coq's CI, this
-variable is usually set to `false` indeed to avoid booting a useless
-job.
+If the `SKIP_DOCKER` Gitlab CI environment variable of a pipeline is not set to `true`,
```

Comment:
Maybe the phrasing "The `SKIP_DOCKER` variable controls whether a Docker image is generated for every CI build" would be a bit better than the current negation mess?

Reply:
The specification is exact this way, though... I'm not the one who gave the current semantics to the CI scripts.

Reply:
I would have written it as `the CI builds a docker image unless SKIP_DOCKER=true` instead of `if SKIP_DOCKER!=true then CI builds a docker image`

Reply:
That's correct, but this phrasing implicitly entails that building the image is the default behaviour.

Reply:
Building the image is the default behaviour, except specifically on Coq's repository.

Reply:
Mkay. Will tweak.